flang/test/Lower/OpenMP/omp-sections.f90

```diff
@@ -0,0 +1,108 @@
+! This test checks the lowering of OpenMP sections construct with several clauses present
+
+! RUN: bbc -fopenmp -emit-fir %s -o - | \
+! RUN: FileCheck %s --check-prefix=FIRDialect
+
+!FIRDialect: func @_QQmain() {
+!FIRDialect: %{{.*}} = fir.alloca i32 {bindc_name = "count", pinned, uniq_name = "_QFEcount"}
+!FIRDialect: %{{.*}} = fir.alloca i32 {bindc_name = "double_count", pinned, uniq_name = "_QFEdouble_count"}
+!FIRDialect: %[[COUNT:.*]] = fir.address_of(@_QFEcount) : !fir.ref<i32>
+!FIRDialect: %[[DOUBLE_COUNT:.*]] = fir.address_of(@_QFEdouble_count) : !fir.ref<i32>
+!FIRDialect: %[[ETA:.*]] = fir.alloca f32 {bindc_name = "eta", uniq_name = "_QFEeta"}
+!FIRDialect: %[[ALLOCATOR:.*]] = arith.constant 1 : i32
+!FIRDialect: omp.sections allocate(%[[ALLOCATOR]] : i32 -> %[[ETA]] : !fir.ref<f32>) {
+!FIRDialect: omp.section {
+!FIRDialect: {{.*}} = arith.constant 5 : i32
+!FIRDialect: fir.store {{.*}} to {{.*}} : !fir.ref<i32>
+!FIRDialect: {{.*}} = fir.load %2 : !fir.ref<i32>
+!FIRDialect: {{.*}} = fir.load %3 : !fir.ref<i32>
+!FIRDialect: {{.*}} = arith.muli %5, %6 : i32
+!FIRDialect: {{.*}} = fir.convert %7 : (i32) -> f32
```

Comment:
Test still contains SSA values (%2, %3, %5, %6, %7 and more in other places). Please replace all.

Reply:
@kiranchandramohan I have done the changes. I have a question: do these SSA values change over time or across architectures? Or is it a good practice to capture them.

Reply:
You should avoid hard-coding the SSA-value tag names in tests as they can and do change over time. There are no guarantees in that regard.

Reply:
Thanks for the clarification.